test/SlowTests/Issues/RavenDB_16079.cs

```diff
@@ -0,0 +1,101 @@
+﻿using FastTests;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Operations.Indexes;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_16079 : RavenTestBase
+    {
+        public RavenDB_16079(ITestOutputHelper output) : base(output)
```

Comment:
I wonder if Git normalize new line ending during checkout process, so we test X vs X here? Maybe we should replace new line endings on the fly during test execution and then compare?

Reply:
https://github.com/ravendb/ravendb/blob/v5.1/.gitattributes#L5

Reply:
I think that it only normalizes line endings at EOL not in the content?

Reply:
oh you are right - we use " and \r \n inside stings. Ignore my comment then.